packages/url_launcher/url_launcher_android/pubspec.yaml

```diff
@@ -2,7 +2,8 @@ name: url_launcher_android
 description: Android implementation of the url_launcher plugin.
 repository: https://github.com/flutter/packages/tree/main/packages/url_launcher/url_launcher_android
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+url_launcher%22
-version: 6.3.18
+version: 6.3.19
+
 environment:
   sdk: ^3.7.0
   flutter: ">=3.29.0"
```

Comment:
@stuartmorgan-g this is a breaking change. 

this plugin with dropped support for Android 21 shows up for the `flutter upgrade` in a project that has sdk constraint like `sdk: ^3.8.1` (i.e. Flutter 3.32.8).

This plugin sdk constraint probably should be bumped to `^3.9.0`